### ISSUE TITLE
librex browser search integration - firefox with LibreX Docker container

### ISSUE THREAD

**Issue by myobfee** [hnhx/librex#248]
In utilizing right-click option to add local LibreX instance as default browser search engine, any subsequent browser searches fail as LibreX defaults to docker IP/port (127.0.0.1:8080) instead of proxied domain for browser bar searches.  Using Cosmos as proxy front-end: https://github.com/azukaar/Cosmos-Server.  LibreX works when accessed via external URL directly.

Hoping there is an un-documented environment variable to add to the docker-compose.yml to address so external domain is used in browser search query instead of docker IP/port (127.0.0.1:8080)

**Comment by elmogard** [comment]
I faced the same issue with Firefox. You can solve this creating a new opensearch.xml file or extracting it from your container and change it to point to your domain. Finally, mount this .xml on your compose file and recreate the container.

      - /volume3/docker/librex/nginx:/var/log/nginx
      - /volume3/docker/librex/logs:/var/log/php7
      - /volume3/docker/librex/opensearch.xml:/var/www/html/opensearch.xml

Regards

**Comment by kubo6472** [comment]
Yes, I would love to have a env var. I think that's what `Change opensearch.xml to point to your domain` means [here](https://github.com/hnhx/librex/wiki/How-to-host-LibreX).
